lib/rollbax/item.ex

```diff
@@ -15,20 +15,71 @@ defmodule Rollbax.Item do
     }
   end
 
-  def compose(draft, {level, msg, time, meta}) do
+  def compose(draft, {level, timestamp, body, meta}) do
     {occurr_data, meta} =
       Map.pop(meta, :rollbax_occurr_data, %{})
     Map.update!(draft, "data", fn(data) ->
       Map.merge(occurr_data, data)
-      |> put_body(msg)
+      |> Map.put("body", body)
       |> put_custom(meta)
       |> Map.put("level", level)
-      |> Map.put("timestamp", time)
+      |> Map.put("timestamp", timestamp)
     end)
   end
 
-  defp put_body(data, msg) do
-    Map.put(data, "body", %{"message" => %{"body" => msg}})
+  def body_for_message(message) do
```

Comment:
Maybe `message_to_body` and `exception_to_body` (that one reads much better for sure)?